### ISSUE TITLE
How to build and install Python from source on Windows on the command line?

### ISSUE THREAD

**Issue by SamCoutteauHybrid** [python/cpython#121641]
# Documentation

On windows, following the documetation you can build a python using:

```
.\PCbuild\build.bat -e -p x64
```

This produces a directory `arm64` containing a python executable and some other miscellaneous files, but it is missing folders such as `Lib`, `includes`, `DLLs`

This is unlike the unix steps

```
mkdir __build/
cd __build/
../configure ... -prefix=...
make
make install
```

This does seem to produce a full proper python installation.

I did see some documentation on how to make an windows installer.
Are you supposed to build python from source, then create an installer and run it?, This seems strange in the context of a command line install. Or am I missing some other commands?

I would like to automate the process of building and installing python from source (on windows) but the documentation seems lacking on this part.

**Comment by SamCoutteauHybrid** [comment]
Thanks, I've just been able to test this and it does seem like what I was looking for. Shame the documentation is a bit lacking in this front.

**Comment by RogueZamboni** [comment]
This is helpful to me as well!  However - is there an easy way to build it /statically/ under Windows?

**Comment by zooba** [comment]
> is there an easy way to build it /statically/ under Windows?

No, that isn't a supported scenario for us right now. There's some work in python-build-standalone that can help, you might want to check that out.

Closing, since the original question was answered.
